assets/prometheus-k8s/kube-rbac-proxy-secret-federate.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+data: {}
+kind: Secret
+metadata:
+  labels:
+    app.kubernetes.io/part-of: openshift-monitoring
+  name: kube-rbac-proxy
+  namespace: openshift-monitoring
+stringData:
+  config.yaml: |-
+    "authorization":
+      "static":
+      - "path": "/metrics"
+        "resourceRequest": false
+        "user":
+          "name": "system:serviceaccount:openshift-monitoring:prometheus-k8s"
+        "verb": "get"
+      - "path": "/federate"
+        "resourceRequest": false
+        "user":
+          "name": "system:serviceaccount:openshift-monitoring:telemeter-client"
```

Comment:
I like that we use a different username but CMO would need to request a new certificate from the Kube API (similar to https://github.com/openshift/cluster-monitoring-operator/blob/d1179dcd3090bc31e3de08259c0c792a1f3a0061/pkg/operator/operator.go#L380).